README.md

```diff
@@ -1,81 +1,24 @@
-# Getting Started with Create React App
+# NuLeaf Resolution App
+
+## Description
+Feeling down on keeping up with your New Year's Resolutions?  This extrodinary web app will enable you to enter and track or search out additional New Year's Resolutions.  Need additional accountability to follow through on your resolutions? This app will allow you to post to your favorite social media circle to let others know which resolutions you are working on.
+
+## Setting Up
+1. Git clone to local machine
+2. NPM install
+3. In terminal NPM Start 
+  Runs the app in the development mode.\
+  Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+4. In terminal NPM Run Server
```

Comment:
The only steps that are really needed for production are `docker-compose up` after you pull

Reply:
You can keep these other steps for "If you want to run in development mode", just be sure to add a "start postgres db" step